app/views/shared/_login_with_google_button.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class ='google-button'>
+    <img width="20px" style="margin-bottom:3px; margin-right:5px" alt="Google sign-in" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Google_%22G%22_Logo.svg/512px-Google_%22G%22_Logo.svg.png" />
+
+    <%= button_to 'Login with Google', '/auth/google_oauth2' %>
+</div>
```

Comment:
All this could be formatted to look better, but Caleb will be doing all that later so I'm leaving it be for now.